Makefile

```diff
@@ -127,11 +127,9 @@ controller-gen:
 ifeq (, $(shell which controller-gen))
 	@{ \
 	set -e ;\
-	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
-	cd $$CONTROLLER_GEN_TMP_DIR ;\
-	go mod init tmp ;\
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.5 ;\
-	rm -rf $$CONTROLLER_GEN_TMP_DIR ;\
+	cd vendor/sigs.k8s.io/controller-tools/cmd/controller-gen;\
+	go install;\
+	cd -;\
 	}
 CONTROLLER_GEN=$(GOBIN)/controller-gen
```

Comment:
Do we need to generate the manifests on the fly? We do not do this for other drivers and we expect all the generated code to be checked in. I was thinking we do this instead:

```diff
diff --git a/Makefile b/Makefile
index 9eca97be..90facf5e 100644
--- a/Makefile
+++ b/Makefile
@@ -138,7 +138,7 @@ else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
 
-$(SYNCER_BIN): $(SYNCER_BIN_SRCS) syncer_manifest
+$(SYNCER_BIN): $(SYNCER_BIN_SRCS)
        CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH) go build $(GOFLAGS_VENDOR) -ldflags '$(LDFLAGS_SYNCER)' -o $(abspath $@) $<
        @touch $@
 
```

Reply:
That looks much better.

I see that's how we used to do (8fe3d6887bc21ddcd418f1cc85b4680f5e728b1c), but I think we forgot to carry this change after our last "rebase".